linux/9.3.x/sitecore-xp-sql/boot.sh

```diff
@@ -25,6 +25,27 @@ echo "### SQL Server PID = '$SQL_PID'."
 
 /opt/attach-databases.sh $dataDir
 
+# set shard
+
+echo "### INIT XCONNECT = '$SQL_PID'."
+
+# to-do
+DatabasePrefix='Sitecore'
+
+/opt/mssql-tools/bin/sqlcmd -S . -U sa -P $SA_PASSWORD -t 120 -l 120 \
+-Q "EXEC sp_MSforeachdb 'IF charindex(''Sitecore'', ''?'' ) = 1 BEGIN EXEC [?]..sp_changedbowner ''sa'' END'"
+
+/opt/mssql-tools/bin/sqlcmd -S . -U sa -P $SA_PASSWORD -t 120 -l 120 \
+-Q "UPDATE [$DatabasePrefix.Xdb.Collection.ShardMapManager].[__ShardManagement].[ShardsGlobal] SET ServerName = '$SQL_HOSTNAME'"
```

Comment:
Should we set this environment variable to something default in the Dockerfile as we do on Windows?